static/app/views/settings/projectMetrics/metricsExtractionRuleForm.tsx

```diff
@@ -46,21 +46,85 @@ interface Props extends Omit<FormProps, 'onSubmit'> {
 }
 
 const HIGH_CARDINALITY_TAGS = new Set([
```

Comment:
There are a few more fields we can take from the typed out [`SpanData`](https://github.com/getsentry/relay/blob/152335119ad6acf10ac69ab853da3f8d27bd47b4/relay-event-schema/src/protocol/span.rs#L191) struct:

- `ai.total_tokens.used`
- `ai.prompt_tokens.used`
- `ai.completion_tokens.used`
- `http.decoded_response_content_length`
- `http.response_transfer_size`
- `cache.key`
- `ai.input_messages`
- `thread.id`
- `sentry.frames.slow`
- `sentry.frames.frozen`
- `sentry.frames.total`
- `frames.delay`
- `user_agent.original`
- `url.full`
- `client.address`

These are the JSON field names in the payload, some of these might not have a constant defined yet.